spec/report_spec.rb

```diff
@@ -1051,11 +1052,11 @@ def gloops
       expect(exception["errorClass"]).to eq("RuntimeError")
       expect(exception["message"]).to eq("'nil' was notified as an exception")
 
-      stacktrace = exception["stacktrace"][0]
-      expect(stacktrace["lineNumber"]).to eq(1047)
+      stacktrace = get_project_frame(exception["stacktrace"])
```

Comment:
This should be moved into a separate test - as this test is just testing the message. We should probably test that the stacktrace is bugsnag initially then this file after the right number of frames. But that might be covered by what is in the stacktrace spec now?